src/main/java/org/sagebionetworks/web/client/widget/entity/browse/EntityFinderWidget.java

```diff
@@ -3,17 +3,18 @@
 import java.util.List;
 
 import org.sagebionetworks.repo.model.Reference;
+import org.sagebionetworks.web.client.jsinterop.EntityFinderScope;
 
-public interface EntityFinder {
+public interface EntityFinderWidget {
```

Comment:
I had to rename EntityFinder -> EntityFinderWidget or else it collided with SRC.SynapseComponents.EntityFinder